apps/widalarmeta/widget.js

```diff
@@ -45,7 +47,7 @@
   function draw(_w, fromInterval) {
 
     // If only show on clock and not on clock
-    if (config.whenToShow === 1 && !Bangle.CLOCK) {
+    if (!!config.whenToShow && !Bangle.CLOCK) {
```

Comment:
In fact we don't even need the `!!` here, the truthiness of `1` / falsiness of `0` already work

```suggestion
    if (config.whenToShow && !Bangle.CLOCK) {
```